src/EditorFeatures/Test2/FindReferences/FindReferencesTests.PropertySymbols.vb

```diff
@@ -1558,5 +1558,32 @@ class P
 </Workspace>
             Await TestAPIAndFeature(input, kind, host)
         End Function
+
+        <WorkItem("http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/538886")>
+        <WpfTheory, CombinatorialData>
+        Public Async Function TestCSharp_PartialProperty1(kind As TestKind, host As TestHost) As Task
```

Comment:
@CyrusNajmabadi: this new Find References test fails only in 1 of the 4 configurations: `TestCSharp_PartialProperty1(kind: API, host: OutOfProcess)`

Do you happen to know--is this failure a sign of something specific that I forgot to update?

Failure message:
```cs
​Expected: [112..116), [161..165)
Actual: [112..116)


using System;
namespace ConsoleApplication22
{
class Program
{
public static partial int {|Definition:Prop|} { get; }
public static partial int Prop => 1;


    static void Main(string[] args)
    {
        int temp = Program.Prop;
    }
}

}


Expected: True
Actual:   False
```

Reply:
looking.

Reply:
Fixed this.